typescript/infra/config/environments/mainnet3/warp/configGetters/getEthereumSeiPumpBTCWarpConfig.ts

```diff
@@ -0,0 +1,42 @@
+import { ethers } from 'ethers';
+
+import {
+  ChainMap,
+  HypTokenRouterConfig,
+  OwnableConfig,
+  TokenType,
+} from '@hyperlane-xyz/sdk';
+
+import { getOwnerConfigForAddress } from '../../../../../src/config/environment.js';
+import {
+  RouterConfigWithoutOwner,
+  tokens,
+} from '../../../../../src/config/warp.js';
+
+const ethereumOwner = '0xa7ECcdb9Be08178f896c26b7BbD8C3D4E844d9Ba';
+const seiOwner = '0xa7ECcdb9Be08178f896c26b7BbD8C3D4E844d9Ba';
+
+export const getEthereumSeiPumpBTCWarpConfig = async (
+  routerConfig: ChainMap<RouterConfigWithoutOwner>,
+  _abacusWorksEnvOwnerConfig: ChainMap<OwnableConfig>,
+): Promise<ChainMap<HypTokenRouterConfig>> => {
+  const ethereum: HypTokenRouterConfig = {
+    ...routerConfig.ethereum,
+    ...getOwnerConfigForAddress(ethereumOwner),
```

Comment:
Is this necessary? My understanding is that if we just set `owner: ethereumOwner` is sufficient and preferred @aroralanuk @tkporter

Reply:
aren't we transferring ownership to PumpBTC before handing it over?

Reply:
this is the practice I was encouraging to ensure that proxyAdmin ownership is also explicitly set

It's a little redundant now that the `routerConfig` omits the `owner` and `ownerOverrides` now, but there was one scenario where we previously set `owner: ethereumOwner` and unintentionally had some `ownerOverrides` in this object that ended up ignoring the `owner` param. This is when I accidentally had transferred ownership of a proxy admin for a warp route

Happy to change the practice but probably makes sense to keep things consistent atm and use this